sdk/src/main/java/io/customer/sdk/hooks/HooksManager.kt

```diff
@@ -0,0 +1,29 @@
+package io.customer.sdk.hooks
+
+interface HooksManager {
+    fun add(subscriber: ModuleHookProvider)
+    fun onHookUpdate(hook: ModuleHook)
+}
+
+internal class CioHooksManager : HooksManager {
+
+    val list: MutableList<ModuleHookProvider> = mutableListOf()
```

Comment:
Instead of a List, I recommend using a Map<String, ModuleHookProvider> to avoid having 2+ hooks per module registered. We don't want to send 2+ events to the same module in the SDK.